app/templates/partials/summary/summary.html

```diff
@@ -1,28 +1,30 @@
 <div class="summary">
-	{%- for group in content.summary.groups -%}
-        {%- if group.title -%}
-            <h2 class="summary__title neptune" id="{{ group.id }}">{{ group.title }}</h2>
-        {%- endif %}
+    {%- if metadata.form_type == '0005' and content.summary.summary_type == 'Summary' -%}
```

Comment:
Is there no better way to decide if the `final-summary.html` should be used?

Reply:
We shouldn't be hardcoding survey specific logic. Why do you need the `== '0005'`?. `Summary` should be the final section and `SectionSummary` for the individual sections.

What am I missing?

Reply:
The reason this is here for now is because the card says we only wanted it for mwss, and if it was used as is on the other surveys it would break, I'd say the 'Summary' type would probably be changed and then we can change some of the logic then. But yeah I agree I don't like it done this way, couldn't think of another for now

Reply:
In the schema i think you just remove the block type for summary and use a final summary block type?